packages/edit-site/src/components/header/style.scss

```diff
@@ -1,3 +1,5 @@
+@import "./more-menu/style.scss";
```

Comment:
We try to import things in the main style.scss of the package for consistency.

Reply:
Updated in 5582081785d77a9e67a1a1a746af76e0ac59200c.